dataprofiler/tests/profilers/test_base_column_options.py

```diff
@@ -0,0 +1,58 @@
+import os
+import unittest
+from unittest import mock
+
+import pandas as pd
+
+from dataprofiler import Data, ProfilerOptions, Profiler
+from dataprofiler.profilers.profiler_options import BaseColumnOptions
+from dataprofiler.tests.profilers.test_boolean_option import TestBooleanOption
+
+
+@mock.patch('dataprofiler.profilers.data_labeler_column_profile.'
+            'DataLabelerColumn.update', return_value=None)
+@mock.patch('dataprofiler.profilers.data_labeler_column_profile.DataLabeler')
+class TestBaseColumnOptions(TestBooleanOption):
+    
```

Comment:
Which option class? I am a little unclear on what is being referenced

Reply:
I guess it isn't necessary to move it, but technically we could get rid of 

```python
@classmethod
def setUpClass(cls):
    cls.option_class = BaseColumnOptions
```

and just do 

```python
class ...:
    option_class = BaseColumnOptions
    .
    .
    .
```

which will be less code in the long run for someone to type

Reply:
I think that might defeat the purpose of the check in the abstract class. TestBaseColumnOptions would just inherit option_class from BooleanOption and use the BooleanOption class for its tests. If we enforce using the setUpClass method (which is generally included because of unittest), then if TestBaseColumnOptions does not set option_class it would default to None.